### PR TITLE
Removing instances of "Framework" and Cleaning unused methods

### LINKED ISSUE #485: Remove .NET Framework isms
I was just looking through the code-base and did a search on ".NET Framework".  Here are a few hits I found:

* [ResXDataNode.cs](https://github.com/dotnet/winforms/blob/15cda680cff86c1d7654f8dd675249733d002b94/src/System.Windows.Forms/src/System/Resources/ResXDataNode.cs#L803)
* [PropertyDescriptorGridEntry.cs](https://github.com/dotnet/winforms/blob/784d45b66d34ca1f153559b7e6dbd0d20fa95c25/src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/PropertyDescriptorGridEntry.cs#L1016)
* [AssertExtensions.cs](https://github.com/dotnet/winforms/blob/0f2036a5f45740be4718da76cf7bf247140f8864/src/Common/tests/InternalUtilitiesForTests/src/AssertExtensions.cs#L15)

I was wondering if there was going to be a structured effort to remove important .NET Framework-isms. The ones I included are all of different kinds. It would be good to see a statement (and maybe this already exists) on what kinds of .NET Framework isms should be removed/cleanedup before shipping .NET Core 3.0.

### PR DESCRIPTION
**Pull request by dreddy-work**
Cleaning DpiHelper class methods that are not used anymore. 
Removing the "Framework" instances. 
Cleaning comments and updating resx templates.

Contributes to #485

### CODE REVIEW

**Comment by RussKie** [comment]
Please submit "Framework" removal changes as a separate commit under #485
I have an open PR https://github.com/dotnet/winforms/pull/1139 but it is probably better to apply your changes separately

File: src/System.Windows.Forms/src/Resources/SR.resx [hunk 1]
```diff
@@ -3158,14 +3158,14 @@ just-in-time (JIT) debugging instead of this dialog box.
     <value>&amp;Quit</value>
   </data>
   <data name="ExDlgSecurityContinueErrorText" xml:space="preserve">
```
Comment (hughbe) [inline src/System.Windows.Forms/src/Resources/SR.resx]: This seems like it should be removed

**Comment by dreddy-work** [comment]
Regarding separate PR, This is all cleanup. low risk and unused code. If you do not have a strong opinion on splitting this, i like to merge this in a single PR.

File: src/System.Windows.Forms/src/Resources/SR.resx (hunk 1 above)
Comment (dreddy-work) [reply src/System.Windows.Forms/src/Resources/SR.resx]: Are you saying this is not used anymore? I have not checked its usage yet. If this is indeed need to be removed, Lets do it in a separate PR. Do not want to block this PR on this.

**Pull request merged**